gym/dev_wrappers/torch_wrapper.py

```diff
@@ -1,14 +1,19 @@
-from typing import Optional, Tuple, Union
```

Comment:
Could you put the copyright here in a top docstring, the text below is good. This is good  https://github.com/openai/gym/blob/master/gym/utils/env_checker.py

Reply:
Could you check on the changes I made to address this? I moved the entire comment minus the args to a docstring at the top of the file but that may be too much? Maybe move the description of the class back into the class' docstring?